doc/mgr/orchestrator.rst

```diff
@@ -575,6 +575,18 @@ Many service specifications can be applied at once using
       all: true
     EOF
 
+Retrieving the running Service Specification
+--------------------------------------------
+
+If started via ``ceph orch apply <rgw|mds|...>`` changing the Service
+Specification directly is normally a bit more complicated. It is easier to
+export the running Service Specification like so::
```

Comment:
@zdover23 can we rephrase this to sound a bit less German?

Reply:
School-english...

```
If services have been started using ``ceph orch apply ...`` without specification file changing it is still possible. An easy way to do so is exporting the running Service Specification::
```

Reply:
I suggest this:

BEGIN QUOTED TEXT
If the services have been started via ``ceph orch apply...``, then directly changing the Services Specification is complicated. Instead of attempting to directly change the Services Specification, we suggest exporting the running Service Specification by following these instructions:
END QUOTED TEXT

Reply:
Ok, thank you @zdover23. The change has been applied.